agent/config/runtime.go

```diff
@@ -835,7 +835,7 @@ type RuntimeConfig struct {
 
 	// PrimaryGateways is a list of addresses and/or go-discover expressions to
 	// discovery the mesh gateways in the primary datacenter. See
-	// https://www.consul.io/docs/agent/options.html#cloud-auto-joining for
+	// https://www.consul.io/docs/agent/config/cli-flags.html#cloud-auto-joining for
```

Comment:
Found some more references to the old page in some go files.. @dnephin it might be good to get more 👀 on these!

Reply:
It's always good to update these, but since we have redirects in place now, I think it's ok if we miss some too.